doc/man3/BIO_s_mem.pod

```diff
@@ -113,9 +113,19 @@ Extract the BUF_MEM structure from a memory BIO and then free up the BIO:
  BIO_set_close(mem, BIO_NOCLOSE); /* So BIO_free() leaves BUF_MEM alone */
  BIO_free(mem);
 
+=head1 RETURN VALUES
+
+BIO_s_mem() and BIO_s_secmem() return the memory BIO method function, which
+is a vaild B<BIO_METHOD> structure.
```

Comment:
I realize that "BIO method function" is taken from description that was there earlier, but I find the term confusing. I mean it doesn't actually return a function, but rather a dispatch table. Thoughts?

Reply:
How about just remove "function":
```
memroy BIO method, a structure of function pointers
```

Reply:
That structure contains other data than function pointers...

Reply:
It might be better by just saying 'return a valid memory B<BIO_METHOD> structure'...